ktor-utils/posix/src/io/ktor/util/CryptoNative.kt

```diff
@@ -23,4 +23,6 @@ public actual fun Digest(name: String): Digest = error("[Digest] is not supporte
 /**
  * Compute SHA-1 hash for the specified [bytes]
  */
-public actual fun sha1(bytes: ByteArray): ByteArray = Sha1().digest()
+public actual fun sha1(bytes: ByteArray): ByteArray = Sha1().apply {
+    update(bytes)
+}.digest()
```

Comment:
I think there is already an extension called `digest` that takes `bytes` directly and does both the `update` and `digest` calls.

Reply:
thanks, fixed